AM_HW9_Notebook.ipynb

```diff
@@ -0,0 +1,461 @@
+{
```

Comment:
<p>That's your acceleration on the spherical theta, phi plane of the earth. Your r acceleration is hopefully much bigger (or much smaller, depending on the direction). But since you are using the tunnel your r acceleration is actually constantly changing. Cartesian coordinates in spherical symmetries is a lot of math.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_Calculus/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>